src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -663,18 +663,17 @@ public void processWindowMessages()
      */
     public void exitJava(int exitCode, boolean hardExit)
     {
-        FMLLog.log.info("Java has been asked to exit (code {}) by {}.", exitCode, Thread.currentThread().getStackTrace()[1]);
+        FMLLog.log.warn("Java has been asked to exit (code {})", exitCode);
         if (hardExit)
         {
-            FMLLog.log.info("This is an abortive exit and could cause world corruption or other things");
+            FMLLog.log.warn("This is an abortive exit and could cause world corruption or other things");
         }
-        if (Boolean.parseBoolean(System.getProperty("fml.debugExit", "false")))
```

Comment:
Why do you remove this system property?

Reply:
read the other code comment

Reply:
I removed it because the only difference between the old behavior with the property enabled and the new behavior is that the top two frames aren't printed now. And, as I stated in the code comment, those aren't relevant for anything.